src/Hazelcast.Net/DistributedObjects/IDistributedObject.cs

```diff
@@ -20,36 +20,36 @@ namespace Hazelcast.DistributedObjects
     /// <summary>
     /// Defines the base contract for all Hazelcast distributed objects.
     /// </summary>
-    public interface IDistributedObject : IAsyncDisposable
```

Comment:
Why the indentation change in this file?

Reply:
left over, will fix it